ush/sub_wcoss2

```diff
@@ -115,6 +115,7 @@ echo "#PBS -j oe"                          >> $cfile
 echo "#PBS -A "$accnt                      >> $cfile
 
 echo "" >> $cfile
+echo "set -x" >> $cfile 
```

Comment:
This debug `set -x` should probably be removed.

Reply:
`set -x` removed at [7cb76d8](https://github.com/NOAA-EMC/GSI/pull/571/commits/7cb76d8b5e0bdd82b333eab789f7109d51336595)